blogposts/2020/announcing-sourcegraph-3.16.md

```diff
@@ -0,0 +1,249 @@
+
+<div class="alert alert-info">
+  <p>
+    <strong>LEARN MORE:</strong><br />
+    <a href="https://info.sourcegraph.com/online-meetup-may21?utm_source=blog">
+      Online meetup: Sourcegraph 3.16 - Thursday, May 21 at 10AM PDT
+    </a>
+  </p>
+</div>
+
+## Introducing version contexts to search past releases
+
+TODO: video
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://www.youtube.com/embed/{VIDEO_ID}" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://youtu.be/{VIDEO_ID}" target="_blank">View on YouTube</a></p>
+</p>
+-->
+
+Many large organizations have old versions of code running in production and need to search across all the code for that release. Sourcegraph 3.16 introduces the ability to create a context consisting of many repositories with specific revisions to be searched. As you explore search results, you might need to navigate your code. Version contexts are respected by code intelligence hovers (go to definition and find references), and allow you to quickly traverse the specific code revisions you care about.
+
+In this release, any non-master branches that are specified in the context will use our non-indexed search. This means that results may be slow for larger deployments. We are working on adding support for indexing additional branches that are specified within the context, and will include this in a future release.
+
+To take advantage of of this feature, add a `versionContexts` block to the experimental features in your site configuration:
+
+```json
+"experimentalFeatures": [
+  "versionContexts": [
+    {
+      "name": "srcgraph 3.16",
+      "revisions": [
+        {
+          "repo": "github.com/sourcegraph/sourcegraph",
+          "ref": "v3.16.0"
+        },
+        {
+          "repo": "github.com/sourcegraph/zoekt",
+          "ref": "b56036a3b745033badc48807bd67e91cd8d73bdf"
+        }
+      ]
+    }
+  ]
+]
+```
+
+## Campaigns
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://www.youtube.com/embed/aqcCrqRB17w" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://youtu.be/aqcCrqRB17w" target="_blank">View on YouTube</a></p>
+</p>
+
+Campaign changeset usage counts including changesets created, added, and merged will be sent back in pings. [#10591](https://github.com/sourcegraph/sourcegraph/pull/10591)
+
+## Best in class syntax highlighting
+
+TODO: Erik is creating screenshots
```

Comment:
@eseliger to add screenshots for syntax highlighting

Reply:
<img width="1147" alt="SourcegraphDark" src="https://user-images.githubusercontent.com/19534377/82244479-9a5e0b80-9941-11ea-838d-ad90ec506674.png">
<img width="1147" alt="SourcegraphLight" src="https://user-images.githubusercontent.com/19534377/82244493-9d58fc00-9941-11ea-936f-3483cee8aabf.png">
<img width="1022" alt="GitHub" src="https://user-images.githubusercontent.com/19534377/82244495-9e8a2900-9941-11ea-9b4f-f0c07c1e4d4d.png">

do you want me to upload them somewhere else? I added the same diff on our end both light and dark (I don't know any other major codehost that supports that!)
and GitHubs highlighting (which is really plain in this particular case - but there are examples where it isn't _that_ plain at least. But overall I think our light theme is much more diverse in colors and accounts much better for the semantic meaning of the tokens highlighted.